README.adoc

```diff
@@ -2051,118 +2063,100 @@ Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 
 // Starting and preparing your cluster for deployment
 ifndef::cloud-hosted[]
-:kube: Kubernetes
-include::{common-includes}/kube-start.adoc[]
+== Starting and preparing your cluster for deployment
+
+If you are using Linux, you can continue this section.
+
+Start your Kubernetes cluster.
+
+Run the following command from a command-line session:
+
+[role=command]
+```
+minikube start
+```
+
+Next, validate that you have a healthy {kube} environment by running the following command from the active command-line session.
```

Comment:
Should we be saying " healthy {kube} environment " or " healthy kubernetes environment " ?

Reply:
I updated "{kube}" to "Kubernetes".